server/views/components/primary-nav/template.njk

```diff
@@ -0,0 +1,46 @@
+<div class="moj-primary-navigation">
+
+    <div class="moj-primary-navigation__container">
+
+        <div class="moj-primary-navigation__nav">
+
+            <nav class="moj-primary-navigation" aria-label="Primary navigation">
+
+                <ul class="moj-primary-navigation__list">
+
+                    <li class="moj-primary-navigation__item">
+                        <a class="moj-primary-navigation__link" aria-current="page"
+                           href="/"> {% if params.data.allocations %}
```

Comment:
Please can we have a test for this if statement?

Reply:
Thanks - I can see a test for when there aren't any unallocated cases now. Can we have one for the other branch of the if statement - when there are some unallocated cases please?

Reply:
This is one reason why I am not keen on PRs! A 30 second conversation can become hours of back and forth commenting